spec/classes/foreman_tasks_service_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+describe 'foreman::tasks_service' do
+  let :facts do
+    on_supported_os['redhat-7-x86_64']
```

Comment:
Given this code is relevant for every distro where we run Foreman, I'd feel more comfortable if this was tested on more platforms. We've already seen that there are subtle problems with dependencies if we take these shortcuts.